locales/hu_HU/LC_MESSAGES/duckduckgo.po

```diff
@@ -1740,28 +1733,28 @@ msgstr ""
 
 msgctxt "SERP footer content"
 msgid "How We Are Profitable"
-msgstr ""
+msgstr "Hogy vagyunk mégis jövedelmezőek"
```

Comment:
@zoli111 Can you double check this? Translating "Hogy vagyunk mégis jövedelmezőek" back to English I get "How profitable we are", which implies "how much" rather than "in what way". Thanks!